lib/services/samples-service.ts

```diff
@@ -122,28 +128,41 @@ export class SamplesService implements ISamplesService {
 					return;
 				}
 
-				outputLines.push(util.format("%s:%s======================", category.name, os.EOL));
+				outputLines.push(util.format("   %s:%s   ======================", category.name, os.EOL));
 
 				_.each(category.samples, (sample: Sample) => {
-					var nameRow = util.format("    Sample: %s", sample.displayName);
-					var descriptionRow = util.format("    Description: %s", sample.description);
-					var gitClone = util.format("    Github repository page: %s", sample.githubUrl);
-					var cloneCommand = util.format("    Clone command: $ appbuilder sample clone %s", sample.name);
+					var nameRow = util.format("      Sample: %s", sample.displayName);
+					var descriptionRow = util.format("      Description: %s", sample.description);
+					var gitClone = util.format("      Github repository page: %s", sample.githubUrl);
+					var cloneCommand = util.format("      Clone command: $ appbuilder sample clone %s", sample.name);
 					outputLines.push([nameRow, descriptionRow, gitClone, cloneCommand].join(os.EOL));
 				});
 			});
 
-			outputLines.unshift("You can choose a sample from the following:");
 			return outputLines.join(os.EOL + os.EOL);
 		}).future<string>()();
 	}
 
-	private getAllSamples(): IFuture<Sample[]> {
+	private getSamples(platform?: string): IFuture<Sample[]> {
 		return (() => {
-			var iceniumOrganizationSamples = this.getRepositories(SamplesService.GITHUB_ICENIUM_LOCATION_ENDPOINT, (repo: any) => SamplesService.GITHUB_REGEX.test(repo.clone_url) && !repo[SamplesService.REMOTE_LOCK_STATE_PRIVATE]).wait();
-			var telerikOrganizationSamples = this.getRepositories(SamplesService.GITHUB_TELERIK_LOCATION_ENDPOINT, (repo: any) => repo.name === SamplesService.NATIVESCRIPT_SAMPLE_CUTENESS_NAME).wait();
-			var repos = _.union(iceniumOrganizationSamples, telerikOrganizationSamples);
+			var regex: RegExp;
+			platform = platform || "";
+			switch(platform.toLowerCase()) {
```

Comment:
You can extract the switch to a separate method. Will make the code more readable :eyeglasses: